backend/test_cache_security.py

```
@@ -0,0 +1,147 @@
+
+    if response.status_code == 200:
+        print("✅ Query parameter authentication accepted")
+    else:
+        print(f"❌ Query parameter authentication rejected: {response.status_code}")
+
+    print(f"Response: {response.text[:200]}...\n")
+
+
+def main():
+    """Run all tests"""
+    print(f"🧪 Testing Cache Cleanup Security")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)